docs/guide/installation/01_linux.md

```diff
@@ -35,7 +35,10 @@ lrwxrwxrwx. 1 root root 13 Oct 19 19:26 usb-Texas_Instruments_TI_CC2531_USB_CDC_
 # Set up Node.js repository and install Node.js + required dependencies
 # NOTE 1: Older i386 hardware can work with [unofficial-builds.nodejs.org](https://unofficial-builds.nodejs.org/download/release/v20.9.0/ e.g. Version 20.9.0 should work.
 # NOTE 2: For Ubuntu see tip below
-sudo curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash -
```

Comment:
The [docs](https://github.com/nodesource/distributions) still mention this installation method, where did you find the `nsolid_setup_deb` stuff?

Reply:
This installation method was on the nodesource ReadMe when I created the PR. And the `https://deb.nodesource.com/setup_xx.x` wasn't on it.

Now, the nodesource ReadMe have been updated to include again the method as you wrote on your ReadMe, and they removed the one I proposed...
The change is on this [PR](https://github.com/nodesource/distributions/commit/55fa99e7bc3c5f1df38d245cf5c25ba3a304828b) and the `#installation-scripts` anchor (on my above link) disappeared...

So I will close this PR.